django/db/models/fields/tuple_lookups.py

```diff
@@ -14,17 +15,28 @@
 from django.db.models.sql.where import AND, OR, WhereNode
 
 
+class TupleField(Field):
+    pass
+
+
 class Tuple(Func):
     function = ""
 
+    def __len__(self):
+        return len(self.source_expressions)
+
+    def __iter__(self):
+        return iter(self.source_expressions)
+
+    @property
+    def output_field(self):
+        return TupleField()
```

Comment:
```suggestion
class Tuple(Func):
    function = ""
    output_field = Field()

    def __len__(self):
        return len(self.source_expressions)

    def __iter__(self):
        return iter(self.source_expressions)
```
Would this be wrong to do? Not sure how I feel about defining a dummy TupleField

Reply:
Yes, I adjusted.

I thought about this, the ideal solution would be to return a `CompositeField`.
`CompositeField` is a field we plan to add after `CompositePrimaryKey`.

Until then, we need some kind of dummy field here. Using `Field` directly is fine I think.

Reply:
> the ideal solution would be to return a CompositeField.
CompositeField is a field we plan to add after CompositePrimaryKey.

makes sense